### PR TITLE
feat(#8): Support a different service.name for the tests

### LINKED ISSUE #8: Support a different service.name for the tests
As far as I see this action in conjunction with https://github.com/inception-health/otel-export-trace-action are reporting to the same service name.

I think it will be handy to let the users to personalise the service name for the test results, therefore the context propagation could be the one in charge to visualise both traces together

What do you think?

### PR DESCRIPTION
**Pull request by pragmaticivan**
fixes #8

### CODE REVIEW

**Comment by pragmaticivan** [comment]
@nikordaris there was no unit tests for the input parts of this action. Probably worth adding some in a new PR

**Comment by pragmaticivan** [comment]
cc @v1v

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "otel-upload-test-artifact-action",
-  "version": "1.1.0",
+  "version": "1.2.2",
```
Comment (pragmaticivan) [inline package-lock.json]: A release was created in main but this version was not updated here.

**Comment by v1v** [comment]
Thanks @pragmaticivan , I'll run some tests in my end between today and tomorrow, and I'll keep you updated with the status, thanks for this!

**Review by nikordaris — DISMISSED** [review]

**Comment by nikordaris** [comment]
@pragmaticivan release-please action is merged in and i think this PR is ready to merge once we fix the conflict

**Comment by pragmaticivan** [comment]
@nikordaris rebased

**Pull request merged**